Modules/_queuemodule.c

```diff
@@ -155,6 +169,9 @@ simplequeue_pop_item(simplequeueobject *self)
 
 /*[clinic input]
 _queue.SimpleQueue.get
+
+    cls: defining_class
```

Comment:
Hmm... will this appear in docstrings and generated docs?

Reply:
No